backend/LexBoxApi/Proxies/FileUploadProxy.cs

```diff
@@ -0,0 +1,76 @@
+using System.Diagnostics;
+using Yarp.ReverseProxy.Forwarder;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authorization;
+using LexBoxApi.Auth.Attributes;
+using LexBoxApi.Auth.Requirements;
+using Microsoft.Extensions.Options;
+using LexCore.Config;
+using LexSyncReverseProxy;
+
+namespace LexBoxApi.Proxies;
+
+public static class FileUploadProxy
+{
+    public const string UserCanUploadMediaFilesPolicy = "UserCanUploadMediaFiles";
+    public const string UserCanDownloadMediaFilesPolicy = "UserCanDownloadMediaFiles";
+
+    public static void AddFileUploadProxy(this IServiceCollection services)
+    {
+        services.AddSingleton<IAuthorizationHandler, MediaFileRequirementHandler>();
+        services.AddHttpContextAccessor();
+        services.AddMemoryCache();
+        services.AddForwarder();
+    }
+
+    public static void MapFileUploadProxy(this IEndpointRouteBuilder app)
```

Comment:
this should return an `IEndpointConventionBuilder` that will require using a group for all these endpoints. Once you do that I'd like to mark them as `WithOpenAPI` so they can be seen from Swagger

Reply:
Done.